test-support/sample-domain-implementation/src/main/java/org/infinispan/protostream/sampledomain/marshallers/AccountMarshaller.java

```diff
@@ -0,0 +1,59 @@
+package org.infinispan.protostream.sampledomain.marshallers;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import org.infinispan.protostream.MessageMarshaller;
+import org.infinispan.protostream.sampledomain.Account;
+
+/**
+ * @author anistor@redhat.com
+ * @deprecated This marshalling mechanism will be removed in 5. Please switch to annotation based marshalling.
+ */
+@Deprecated
+public class AccountMarshaller implements MessageMarshaller<Account> {
```

Comment:
We appear to have these same marshallers in `src/test/java/org/infinispan/client/hotrod/query/testdomain/protobuf/marshallers`. Can we remove those in favour of using these?

Reply:
Actually the sample domain AccountMarshaller is not the same that is present on src/test/java/org/infinispan/client/hotrod/query/testdomain/protobuf/marshallers.
Because they marshall different classes: Account vs AccountPB.
I wouldn't touch it with this contribution,  I would open a follow up to simplify all the query tests.